src/target/common/stm32/clock.c

```diff
@@ -226,3 +226,21 @@ u32 RTC_GetValue()
     return value;
 }
 
+void _usleep(u32 x)
+{
+#if FREQ_MHz == 72
+    #define COUNT 24
+#else
+    #error Need define the count for usleep
+#endif
```

Comment:
How about something like:
```
# if (FREQ_MHZ % 3)
    #error Frequency is not a multiple of 3 - need custom usleep
# else
   #define COUNT (FREQ_MHz / 3)
#endif
```
It probably won't work properly if we ever see an f4, but we can deal with that when the day comes